dts/arm/nordic/nrf52840.dtsi

```diff
@@ -40,7 +40,8 @@
 
 	soc {
 		uart0: uart@40002000 {
-			compatible = "nordic,nrf-uarte", "nordic,nrf-uart";
+			/* uart can be either UART or UARTE, for the user to pick */
+			/* compatible = "nordic,nrf-uarte" or "nordic,nrf-uart"; */
 			reg = <0x40002000 0x1000>;
 			interrupts = <2 1>;
 			status = "disabled";
```

Comment:
@galak where would you put default RAM size for uarte buffer? In this common data or anywhere else?

Reply:
I think it would be a property in uarte node.  So we'd add it to the yaml & .dtsi.  However I do see how that would create a difference between uart and uarte.